server/core/region_storage.go

```diff
@@ -48,20 +52,25 @@ const (
 )
 
 // NewRegionStorage returns a region storage that is used to save regions.
-func NewRegionStorage(ctx context.Context, path string) (*RegionStorage, error) {
+func NewRegionStorage(
+	ctx context.Context,
+	path string,
+	encryptionKeyManager *encryptionkm.KeyManager,
```

Comment:
How about using `opts ...Option` code design? This is a method compatible with the old code.

Reply:
will do